charts/legalhold/templates/secret.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ .Release.Name }}-hold-token"
+type: Opaque
+data:
+  token: "{{ required "must specify serviceToken" .Values.serviceToken  | b64enc }}"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ .Release.Name }}-hold-tls"
+type: kubernetes.io/tls
+data:
+  tls.crt: "{{ required "Must specify tlsCrt" .Values.tlsCrt | b64enc }}"
+  tls.key: "{{ required "Must specify tlsKey" .Values.tlsKey | b64enc }}"
```

Comment:
suggestion: split these into to files (personal pref) or rename file into `secrets.yaml`
issue: add newline

Reply:
It was originally called `secrets.yaml` but we have `secrets.yaml` in our global `.gitignore` as that's the naming convention we use for helm secret values

Reply:
Ah, right/ I remember reading about it in the chat. Splitting it would be my personal preference, but eventually, it's up to you. If you do, prefixing the file with `secret` probably makes sense.

Reply:
I'll split them up in two files